src/index.html

```diff
@@ -7,8 +7,46 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html">
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="logo"
+        >
+      </a>
+      <nav class="nav">
```

Comment:
```suggestion
      </a>
      
      <nav class="nav">
```
don't forget to add spaces between one-level blocks. Fix it all over the page